certified-connectors/Oneflow/apiProperties.json

```diff
@@ -57,7 +57,6 @@
         }
       }
     ],
-    "publisher": "Oneflow AB",
-    "stackOwner": "Oneflow AB"
+    "publisher": "Dmitry Kleshchov"
```

Comment:
@DmitryKleshchov can you please explain why did you remove the `Oneflow AB` from the [publisher and stackOwner properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)?